src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -100,7 +108,7 @@ def test_subtest(self):
         try:
             process.run(cmd, env=env)
         except process.CmdError as result:
-            if result.result.exit_status is not 0:
+            if result.result.exit_status == 0:
```

Comment:
Does this handle the case where the CmdError exception is raised due to an interrupt?

Reply:
I'm not sure TBH, but that's probably a matter for another ticket.  The change above was really only to satisfy pylint.

Reply:
Ok.  Is the logic change intentional?  Should it be `result.result.exit_status != 0`?

Reply:
Never mind.  This is an outdated change.